docs/categories.yml

```diff
@@ -54,6 +54,13 @@ starter:
   - Styling:None
   - Testing
   - Transitions
+  - Apollo
```

Comment:
Please don't add new tags.

Reply:
it was mentioned on Gatsby's docs: https://www.gatsbyjs.org/contributing/submit-to-starter-library/#adding-new-tag

Reply:
While we support adding tags, we strongly discourage that. We have had plenty of tags that only have 1 instance and that made the UI confusing. We've just made a cleanup to remove them. Therefore the correct solution for your starters would be not adding new tags, but removing tags that don't exist from your entries.

Thank you!